estonian_e_invoice/validation/validators.py

```diff
@@ -1,6 +1,81 @@
-from cerberus import Validator
+from datetime import datetime
+from decimal import Decimal
 
+from cerberus import TypeDefinition, Validator
+from estonian_e_invoice.entities import (
+    VAT,
+    AccountInfo,
+    BuyerParty,
+    ContactData,
+    InvoiceInformation,
+    InvoiceItemGroup,
+    InvoiceSumGroup,
+    InvoiceType,
+    ItemDetailInfo,
+    ItemEntry,
+    LegalAddress,
+    PaymentInfo,
+    SellerParty,
+)
 
-class BaseValidator(Validator):
-    def validate_object(self, obj):
-        return self.validate(obj.__dict__)
+DECIMAL_TYPE = TypeDefinition("decimal", (Decimal,), ())
+VAT_TYPE = TypeDefinition("vat", (VAT,), ())
+LEGAL_ADDRESS_TYPE = TypeDefinition("legal_address", (LegalAddress,), ())
+CONTACT_DATA_TYPE = TypeDefinition("contact_data", (ContactData,), ())
+ACCOUNT_INFO_TYPE = TypeDefinition("account_info", (AccountInfo,), ())
+INVOICE_TYPE_TYPE = TypeDefinition("invoice_type", (InvoiceType,), ())
+ITEM_DETAIL_INFO_TYPE = TypeDefinition("item_detail_info", (ItemDetailInfo,), ())
+INVOICE_INFORMATION_TYPE = TypeDefinition(
+    "invoice_information", (InvoiceInformation,), ()
+)
+INVOICE_ITEM_GROUP_TYPE = TypeDefinition("invoice_item_group", (InvoiceItemGroup,), ())
+INVOICE_SUM_GROUP_TYPE = TypeDefinition("invoice_sum_group", (InvoiceSumGroup,), ())
+PAYMENT_INFO_TYPE = TypeDefinition("payment_info", (PaymentInfo,), ())
+ITEM_ENTRY_TYPE = TypeDefinition("item_entry", (ItemEntry,), ())
+INVOICE_PARTY_TYPE = TypeDefinition("invoice_party", (SellerParty, BuyerParty,), ())
+
+
+class CustomValidator(Validator):
+    types_mapping = Validator.types_mapping.copy()
```

Comment:
```suggestion
    types_mapping = {
        **Validator.types_mapping,
        "decimal": DECIMAL_TYPE,
        ...
    }
```

or if this fails to work in python35 

```suggestion
    types_mapping = Validator.types_mapping.copy().update({
        "decimal": DECIMAL_TYPE,
        ...
    }
```